devices/HapticGlove/include/HapticGlove.h

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file HapticGlove.h
+ * @authors  Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef HAPTIC_GLOVE_H
+#define HAPTIC_GLOVE_H
```

Comment:
Can you please format the entire PR with the shipped clang-format file?

Reply:
@kouroshD References for using [clang-format file](https://github.com/robotology/wearables/blob/master/.clang-format) : https://www.vikingsoftware.com/using-clang-format-with-qtcreator/

Reply:
Done, added.